packages/transition-backend/src/services/odTrip/__tests__/odTripProvider.test.ts

```diff
@@ -56,7 +63,12 @@ test('Parse a csv file, 4326 projection, departure time, HMM times', async () =>
             unused: '0500'
         }
     ];
-    currentData = data;
+
+    // Mock a CSV file stream based on the above test values
+    (fs.createReadStream as jest.Mock).mockReturnValue(createCsvStream('id,originX,originY,destinationX,destinationY,time,unused\n' +
```

Comment:
In tests, you can use mockReturnValueOnce instead, to make sure the mocked value is returned only for the next call and the default one is used otherwise.